source/vscode/package.json

```diff
@@ -925,6 +925,25 @@
           ],
           "additionalProperties": false
         }
+      },
+      {
+        "name": "qsharp-get-library-descriptions",
```

Comment:
So since you don't do any filtering, how much data does this end up being? Does it fit comfortably into our token budget, even when the conversation has gotten long? (We have debug logs to make manual testing a bit easier for this).

Reply:
The returned string is 150879 characters long.

Reply:
The token count seems to be consistently lower than the token budget.